### DISCUSSION TITLE
it is ios it is not 100% compatible

### DISCUSSION THREAD

**Discussion by MAFLIXD** [immich-app/immich#10313] [Q&A]
### The bug

iOS phones have charging failures

### The OS that Immich Server is running on

iOS phones have charging failures

### Version of Immich Server

iOS phones have charging failures

### Version of Immich Mobile App

iOS phones have charging failures

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
iOS phones have charging failures
```

### Your .env content

```Shell
iOS phones have charging failures
```

### Reproduction steps

```bash
1.
2.
3.
...
iOS phones have charging failures
```

**Comment by chowhi123** [comment]
I can hardly see a reason why immich would make your phone fail to charge. It could be that your phone is overheating which would be an apple problem. At least to say THAT is not a compatibility issue at all.
